web/README.md

```diff
@@ -1,26 +1,33 @@
 # Web
+[comment]: <> (Double check Americanised vs Anglicised version of spellings throughout all documentation as well.)
+[comment]: <> (Double check consistency with "front-end" vs "front end" throughout documentation.)
+This is the main front-end website project. This project will consume the output of the front-end components project via NPM.
 
-This is the main front-end website project. This project will consume the output of the front end components project via NPM.
-
-This is an MVC web app written in C#. It's main purpose is to provide proxy authentication/authorization services, along with any static pages that need serving
+This is an MVC web app written in C#. It's main purpose is to provide proxy authentication/authorisation services, along with any static pages that need serving
 
 [![Build Status](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_apis/build/status%2FWeb%20CICD?branchName=main)](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_build/latest?definitionId=2600&branchName=main)
 
 ## Getting started
 
 ### Run the Web App locally
-
+[comment]: <> (Should we specify our dependencies here? Or link to the document on how to set up the dependencies?)
 #### Secret storage
 In a console window:
-1. Navigate to `Web.App` project root
+1. Navigate to the `Web.App` project root
 2. Run `dotnet user-secrets init` to initialise secrets in the directory
 
-> Note: If there is already a `<UserSecretsId>` setting in the `Web.App` project file then `dotnet user-secrets init` will fail. This is because the dotnet tool thinks the user secrets has already been initialised. To avoid this run `dotnet user-secrets set "PLACEHOLDER" "PLACEHOLDER". This will create a `secrets.json` file in the folder location described [here](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-8.0&tabs=linux#how-the-secret-manager-tool-works). At this point you can update the `secrets.json` by hand with all of the required settings.
+[comment]: <> (What should be in place of "PLACEHOLDER" here?)
```

Comment:
PLACEHOLDER - it's being used to just create the file, which can then be manually updated, as described.

Reply:
It shouldn't be anything - it any value. It's just a dummy to get the file created.